spring-cloud-gcp-samples/spring-cloud-gcp-sql-mysql-r2dbc-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.cloud.gcp.sql.database-name=[database-name]
+spring.cloud.gcp.sql.instance-connection-name=[instance-connection-name]
+# Leave empty for root, uncomment and fill out if you specified a user
+# spring.r2dbc.username=
```

Comment:
Can we just piggy-back on the `spring.database.username` property? Same for password.

Reply:
In Spring Boot auto-configuration, the connection factory configurations are derived from`[R2dbcProperties`](https://github.com/spring-projects/spring-boot/blob/e38e72132206578ecf3ddaaa07ea3632e670f936/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/r2dbc/ConnectionFactoryConfigurations.java#L58) so it may not be possible to replace `spring.r2dbc.*` with `spring.datasource.password`?